index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
```

Comment:
@jyasskin Could you please provide your feedback in [crbug 661478](https://bugs.chromium.org/p/chromium/issues/detail?id=661478&desc=2#c23), @noncombatant made user gesture as one of the requirements for accessing sensor data. We need to have consensus before we add such information to the spec.